qt_gui_cpp/src/qt_gui_cpp/CMakeLists.txt

```diff
@@ -29,6 +30,9 @@ set(qt_gui_cpp_INCLUDE_DIRECTORIES
 set(qt_gui_cpp_LINK_LIBRARIES
   ${QT_QTCORE_LIBRARY}
   ${QT_QTGUI_LIBRARY}
+  ${pluginlib_LIBRARY}
+  ${TinyXM2L_LIBRARY}
+  ${rcutils_LIBRARY}
```

Comment:
All of these variables look bogus to me. `ament_cmake` packages don't export a `*_LIBRARY` variable and the TinyXML2 one has a spelling error.

Reply:
You're write. These were not needed